mp/src/game/shared/neo/neo_player_shared.h

```diff
@@ -311,4 +311,24 @@ struct PlayerXPInfo
 void DMClSortedPlayers(PlayerXPInfo (*pPlayersOrder)[MAX_PLAYERS + 1], int *piTotalPlayers);
 #endif
 
+// NEO TODO: Ubuntu release names
```

Comment:
Editable txt would be nice :^)

Reply:
There's an issue of names being dealt with and done server-side. That could mean each player sends over a whole list of names but that seems much, or rather name displays need to defer more to client-side. So I won't bother with it now.